content/docs/other/a_word_on_backward_compatibility.md

```diff
@@ -18,10 +18,13 @@ This is the current list of feature flags:
 
 | **Flag**                               | **Since** | **Description** |
 |----------------------------------------|-----------|-----------------|
+| no\_task\_chaining\_cmdline\_args      | 1.4       | |
+| task\_chaining\_cmdline\_args          | 1.4       | |
```

Comment:
Those are nice examples of the idea of "feature flags should come in pairs" ([related section](https://github.com/RexOps/Rex/blob/master/CONTRIBUTING.md#backwards-compatibility) of the Rex contributing guide).

The description for them could be "{Dis,En}able per-task argument parsing" ("Disable" for the `no_` prefix, and "Enable" for the other one).

Reply:
Do we have an explanation what "argument parsing" means in this context?

Reply:
As for the naming of the page: agreed, but I'll do that once we fininsh the descriptions. I'm for merging this into the Guides category under the name 'Feature flags' indeed.

Reply:
Argument parsing is "rex argument parsing". Normal case:

```bash
rex --rex --args task1 task2 --task1arg=value --task2arg
```

IIRC, historically, if there are multiple tasks, all tasks get all task arguments (i.e `task1` and `task2` both gets both `task1arg` and `task2arg`). This feature flag makes it possible to pass arguments per task:

```bash
rex --rex --args task1 --task1arg=value task2 --task2arg
```
so `task1` only gets `task1arg` and `task2` only gets `task2arg).

The "task chaining" in the name is a name of a proposed, but I believe not yet fully implemented feature. This task argument parsing behavior was a first step towards that goal.

Page rename: awesome :+1:

Reply:
Incorporated.